types/constants.go

```diff
@@ -3,10 +3,11 @@ package types
 type ContentType string
 
 const (
-	DIDJSON   ContentType = "application/did+json"
-	DIDJSONLD ContentType = "application/did+ld+json"
-	JSONLD    ContentType = "application/ld+json"
-	JSON      ContentType = "application/json"
+	DIDJSON    ContentType = "application/did+json"
+	DIDJSONLD  ContentType = "application/did+ld+json"
+	JSONLD     ContentType = "application/ld+json"
+	JSON       ContentType = "application/json"
+	W3IDDIDURL string      = "https://w3id.org/did-url-dereferencing"
```

Comment:
There's two different headers/profiles but only one listed here, so I think this hasn't been accounted for at all. This is `profile="https://w3id.org/did-resolution"` described in the task, which is different then URL dereferencing for resources.

Reply:
@ankurdotb That is DID Resolution work is in a separate PR by sownak. This is specific to DID URL Resolution